src/main/java/de/uni_hamburg/informatik/swt/se2/mediathek/startup/StartUpMediathek_Blatt06.java

```diff
@@ -70,9 +70,12 @@ private static void erstelleServices()
                     datenEinleser.getMedien());
             _kundenstamm = new KundenstammServiceImpl(
                     datenEinleser.getKunden());
-            _verleihService = new VerleihServiceImpl(_kundenstamm,
-                    _medienbestand, datenEinleser.getVerleihkarten());
-            _vormerkService = new VormerkServiceImpl(_verleihService);
+            VormerkServiceImpl vormerkServiceImpl = new VormerkServiceImpl(null); // temporär null
```

Comment:
Das wird so nicht klappen, in Zeile 59 gibst du die momentane Referenz an VerleihServiceImpl weiter, jedoch setzt du in Zeile _vormerkService auf eine neue Instanz, _service zeigt jedoch noch auf die alte!
Das ist so wie
x = new FalschesX()
y = new Y(x)
x = new RichtigesX()

y zeigt immer noch auf das falsche X!

Reply:
Doch ggf. Tests für StartUpMediathek???
Methode für das Erstellen der Services?